leda/components/AutoComplete/types.ts

```diff
@@ -96,69 +101,67 @@ export interface BlurEvent extends React.FocusEvent<HTMLInputElement> {
 }
 
 export interface AutoCompleteProps<T extends Suggestion = Suggestion> extends ValidationProps {
-  /** Браузерное автозаполнение поля ввода, по умолчанию "off" */
+  /** Browser autofill, off is the default value. Works as HTML autoComplete attribute` */
   autoComplete?: string,
-  /** Сравнение объектов по произвольному полю, а не по ссылке */
+  /** ... */
```

Comment:
Maybe too short)

Reply:
I don't know how it works, so i didn't document this prop. it will be revisited and a proper comment will be added